### PR TITLE
Fix grid input font sizes and border colors

### PR DESCRIPTION
**Pull request by andymcfee**
### OLD
image

### NEW
image

### CODE REVIEW

File: src/alto-ui/Form/TextField/TextField.scss [hunk 1]
```diff
@@ -23,13 +23,13 @@
...
-  &:hover:not(.textfield--readOnly):not([disabled]) {
+  &:hover:not(.textfield--readOnly):not([disabled]):not(.DatagridCellInput) {
     border-color: $blue;
     box-shadow: 0 0 0 1px $blue;
   }
 
-  &:focus:not(.textfield--readOnly),
-  &:focus:hover:not(.textfield--readOnly) {
+  &:focus:not(.textfield--readOnly):not(.DatagridCellInput),
+  &:focus:hover:not(.textfield--readOnly):not(.DatagridCellInput) {
```
Comment (renaudtertrais) [inline src/alto-ui/Form/TextField/TextField.scss]: Cant validate that. We cant make ref to external component. We need to find a better way to solve it.

Comment (andymcfee) [reply src/alto-ui/Form/TextField/TextField.scss]: I removed these refs

File: src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.scss [hunk 2]
```diff
@@ -90,14 +90,15 @@
...
+  border: 1px solid transparent;
   overflow: hidden;
   max-width: 100%;
   flex: 1;
   border-radius: $border-radius-default;
 
   @extend %text-ellipsis;
 
-  &--compact:not(.DatagridCell__content--header) {
+  &--compact:not(.DatagridCell__content--header):not(.DatagridCell__content--editable) {
```
Comment (renaudtertrais) [inline src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.scss]: A bit complex, but ok.

File: src/alto-ui/Datagrid/components/DatagridCellInput/DatagridCellInput.scss [hunk 3]
```diff
@@ -6,18 +6,52 @@
   flex: 1;
   min-width: 0;
 
+  // TODO Refactor this to not nest
```
Comment (renaudtertrais) [inline src/alto-ui/Datagrid/components/DatagridCellInput/DatagridCellInput.scss]: 👍

**Pull request merged**